src/Illuminate/Foundation/Console/RouteCacheCommand.php

```diff
@@ -58,12 +60,28 @@ public function handle()
             return $this->error("Your application doesn't have any routes.");
         }
 
+        $symfonyRoutes = new SymfonyRouteCollection();
+        $attributes = [];
+
         foreach ($routes as $route) {
             $route->prepareForSerialization();
+
+            $name = $route->getName() ?? Str::random();
```

Comment:
For the new system I needed every route to have a name so if a route doesn't has a name, I generate one for it. I don't know if this has complications for any existing app. Does anyone have any thoughts if this could affect some apps?